### PR TITLE
Updates dependencies

### PR DESCRIPTION
**Pull request by kelsos**
I updated the dependencies to the latest versions, most of them are development except electron that has been bumped to 8.2.1 and a vuetify patch update.

Prettier, the formatter had a major update with some opinionated changes that are not configurable. I kept whatever I could configure as before https://prettier.io/blog/2020/03/21/2.0.0.html

### CODE REVIEW

**Comment by LefterisJP** [comment]
This breaks the frontend Travis-ci. Are you sure it does not break other stuff? Perhaps wait until after v1.4.0 with this? So include it in v1.5.0?

**Comment by kelsos** [comment]
> This breaks the frontend Travis-ci. Are you sure it does not break other stuff? Perhaps wait until after v1.4.0 with this? So include it in v1.5.0?

Nope, it doesn't break anything. I had a forgotten variable in eslint.js when I split the commits.

I did a full check on the followup commit that will come on a new PR. 
unit tests/integration tests/serve/package

The electron update is minor btw

**Comment by kelsos** [comment]
Let me move some more changes from the other PR to this one

**Comment by LefterisJP** [comment]
Okay I trust you. But damn those automatic linters: 
```diff
+13,161 −9,064
```

**Comment by kelsos** [comment]
@LefterisJP and the other will be bigish since it introduces a lint rule for import ordering

**Comment by LefterisJP** [comment]
I don't like meaningless huge diffs. If possible please stick to one set of linting rules.

**Comment by kelsos** [comment]
@LefterisJP the problem is that currently, we have no import ordering rule, so the ordering is inconsistent. If there was another way to introduce a rule without many changes I would do it. 

Also, this PR is actually that big because of the dependency update. The actually automated lint fixes are around 100 lines. The other 13000 is the `package-lock.json`. There is no easy way around that, unfortunately.

**Comment by LefterisJP** [comment]
> The other 13000 is the package-lock.json

hahhhhahahahaha

node ...

File: .travis.yml [hunk 1]
```diff
@@ -83,6 +83,7 @@ cache:
     - $HOME/.cache/.rotkehlchen-test-dir
     - $HOME/.ethash
     - $HOME/.bin
+    - $HOME/.cache/Cypress
```
Comment (kelsos) [inline .travis.yml:86]: @LefterisJP Added this to save some time with Cypress

Comment (LefterisJP) [reply .travis.yml:86]: nice thanks! Does Github actions also have this?

Comment (kelsos) [reply .travis.yml:86]: Yes I first added it there

**Comment by LefterisJP** [comment]
A good thing is that it seems Github actions finish faster than Travis :)

**Comment by kelsos** [comment]
They are a bit faster yes

**Comment by LefterisJP** [comment]
Not just a bit. I see ~10 mins difference. ~5 mins in the OSX VM of Travis which is realy slow. But also Travis has the UI tests that run. Github actions don't yet?

**Comment by kelsos** [comment]
I am not sure if the comment confused it. It should be there but somehow the pattern matching didn't work. I had it tested on my fork and it worked

**Comment by kelsos** [comment]
Btw I nuked the `package-lock.json` and did an `npm install` now the diff is smaller...

**Comment by LefterisJP** [comment]
> I am not sure if the comment confused it. It should be there but somehow the pattern matching didn't work. I had it tested on my fork and it worked

you mean Github actions should have also included a frontend test?

**Comment by kelsos** [comment]
> > I am not sure if the comment confused it. It should be there but somehow the pattern matching didn't work. I had it tested on my fork and it worked
> 
> you mean Github actions should have also included a frontend test?

Yes after the test-backend it should run an integration-test job but it didn't

**Comment by LefterisJP** [comment]
> Yes after the test-backend it should run an integration-test job but it didn't

Have you tested it works normally? The extra job I mean

**Comment by kelsos** [comment]
> > Yes after the test-backend it should run an integration-test job but it didn't
> 
> Have you tested it works normally? The extra job I mean

Yes I did test it on my fork

**Pull request merged**